docs/NNNN-direct-clean-path-layout.md

```diff
@@ -0,0 +1,444 @@
+# OCFL Community Extension NNNN: Direct Clean Path Layout
+
+* **Extension Name:** NNNN-direct-clean-path-layout
+* **Authors:** Jürgen Enge (Basel)
+* **Minimum OCFL Version:** 1.0
+* **OCFL Community Extensions Version:** 1.0
+* **Obsoletes:** n/a
+* **Obsoleted by:** n/a
+
+## Overview
+
+This extension is intended to be used to map logical paths to safe content paths.
+This is done by replacing or removing "dangerous characters" from names.
+
+This functionality is based on David A. Wheeler's essay "Fixing Unix/Linux/POSIX Filenames" (https://www.dwheeler.com/essays/fixing-unix-linux-filenames.html)
+
+### Usage Scenario
+
+This extension is intended to be used when you want the internal OCFL layout to match
```

Comment:
Now that this extension is no longer targeting storage layouts, should this sentence be changed to something like:
```
This extension is intended to be used when you want to ensure file and directory names do not include problematic characters.
```